level2/datasources.tf

```diff
@@ -25,3 +25,9 @@ data "aws_ami" "ubuntu" {
   owners = ["aws-marketplace"]
 }
 
+#Data cource for AWS S3 full access policy
+data "aws_iam_policy" "s3-fullaccess-policy" {
+  name = "AmazonS3FullAccess"
+}
+
+
```

Comment:
Please remove the last two lines. The file should end with only a single newline